client/src/pages/Room/Room.tsx

```diff
@@ -51,23 +54,26 @@ const Room: React.FC = () => {
   } = useForm<IInputs>();
 
   const {
-    data: rooms,
+    data: roomData,
     error: fetchRoomError,
     loading: fetchRoomLoading,
     subscribeToMore,
+    fetchMore,
   } = useGetRoomQuery({
-    onCompleted() {
-      scrollToBottom(bodyRef?.current);
+    notifyOnNetworkStatusChange: true,
+    onCompleted(data) {
+      if (!isFetchingMore) {
```

Comment:
keeping track of isFetchingMore because setting "notifyOnNetworkStatusChange: true" would fire "onComplete" whenever "fetchMore" would succeed, resulting the to scroll bottom when user scrolls upwards in order to fetch previous messages